### ISSUE TITLE
Investigate learned wikidata subject typing

### ISSUE THREAD

**Issue by harrisonpim** [wellcometrust/platform#3426]
[This paper](https://arxiv.org/abs/1802.01021) by Raiman and Raiman shows massive improvements to disambiguation accuracy by learning a type system based on the graphy wikidata subject hierarchy. Apparently A 100d binary vector is capable of expressing the full class structure of wikidata. Running a neural network with context vectors etc against targets expressed as these binary vectors should allow us to create a complete end-to-end system for link identification and disambiguation! 

Best results for creation of the binary vector by the authors was achieved using a [genetic algorithm](https://en.wikipedia.org/wiki/Genetic_algorithm), targeted against a proxy objective. 

Blog post [here](https://blog.openai.com/discovering-types-for-entity-disambiguation/), and github repo full of tensorflow code [here](https://github.com/openai/deeptype)
